doc/api/buffer.markdown

```diff
@@ -638,9 +609,9 @@ console.log(buf.length);
 ### buf.readDoubleBE(offset[, noAssert])
 ### buf.readDoubleLE(offset[, noAssert])
 
-* `offset` Number
-* `noAssert` Boolean, Optional, Default: false
-* Return: Number
+* `offset` {Number} `0 <= offset <= buf.length`
```

Comment:
`0 <= offset <= buf.length` is very "techy", but I can't think of a better way to describe this requirement.

Offset of `< 0` or `> byteLength` would `throw new RangeError('Index out of range')`

Maybe a paragraph about this error instead of a "techy" requirement expression?